crates/metaslang/graph_builder/vscode/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I wonder if we need the vscode extension here? I think we discussed removing/not including it?

Reply:
I am intending to enable it